### PR TITLE
chore: Redesign Navbar and Filters

### PR DESCRIPTION
**Pull request by peterpaints**
#### What does this PR do?
- The NavBar and Filters (which only appear in /dashboard) have been redone to resemble Humphrey's designs as closely as possible.
#### Where should the reviewer start?
- Go to /dashboard in localhost
#### Screenshots (if appropriate)
screen shot 2018-05-23 at 5 04 28 pm

### CODE REVIEW

File: src/Components/CustomMenu/CustomMenu.Component.jsx [hunk 1]
```diff
@@ -25,11 +25,20 @@ export class CustomMenu extends React.Component {
...
+      <div className="custom-menu">
         <SelectField
+          underlineStyle={{display: 'none'}}
+          iconStyle={{ fill: '#000000', marginRight: '1rem' }}
+          labelStyle={{ textAlign: 'right' }}
           value={this.state.value}
           onChange={this.handleChange}
-          style={{ display: 'block', fontSize: '0.8rem', textAlign: 'center', width: '10rem', padding: '0 auto' }}
+          style={
+          {
```
Comment (kimobrian) [inline src/Components/CustomMenu/CustomMenu.Component.jsx]: Put this style outside the component in some variable and then use it here.

Comment (peterpaints) [reply src/Components/CustomMenu/CustomMenu.Component.jsx]: @kimobrian Will look into this and make changes ASAP

Comment (peterpaints) [reply src/Components/CustomMenu/CustomMenu.Component.jsx]: @kimobrian I've just researched this. Apparently, for material ui components, one has to style them inline. You can't style them from a separate stylesheet. Unless you mean that I declare a styles variable inside the render but above the return function like this?
screen shot 2018-05-23 at 7 17 01 pm

Comment (kimobrian) [reply src/Components/CustomMenu/CustomMenu.Component.jsx]: Yes, that's what I meant. Declare it as a variable and then use it.

Comment (peterpaints) [reply src/Components/CustomMenu/CustomMenu.Component.jsx]: Done

**Comment by kimobrian** [comment]
I'm not good with design but the UI looks good to me. I will go ahead and merge this.

**Pull request merged**